ci-operator/config/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master.yaml

```diff
@@ -76,12 +76,11 @@ tests:
       CONFIG_TYPE: minimal
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
-- as: e2e-openstack-byon
+- as: e2e-openstack-proxy
   optional: true
-  run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
We should probably have the equivalent `skip_if_only_changed: ^docs/` instruction instead of completely remove the `run_if_changed`.

This would also be nice to add it to `openshift-cluster-api-provider-openstack-release-4.9.yaml` where it's missing.